### PR TITLE
[W5.7][W13-4]Lee Yi Wei, Joel

### PR DESCRIPTION
**Pull request by lywjoel**

### CODE REVIEW

**Comment by ongyz** [comment]
New feature works, everything seems great!

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -79,14 +79,14 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
...
-The search is case sensitive, the order of the keywords does not matter, only the name is searched,
+The search is case insensitive, the order of the keywords does not matter, only the name is searched,
 and persons matching at least one keyword will be returned (i.e. `OR` search).
 ====
 
 Examples:
 
 * `find John` +
-Returns `John Doe` but not `john`.
+Returns `John Doe`.
```
Comment (parvathy-sudhir-pillai) [inline docs/UserGuide.adoc:89]: Good job updating the user doc

File: src/seedu/addressbook/Main.java [hunk 2]
```diff
@@ -111,6 +111,9 @@ private CommandResult executeCommand(Command command)  {
             CommandResult result = command.execute();
             storage.save(addressBook);
             return result;
+        } catch (StorageFile.ReadOnlyException roe) {
+            ui.showToUser("File is read only.");
```
Comment (parvathy-sudhir-pillai) [inline src/seedu/addressbook/Main.java:115]: Better to avoid these magic phrases and declare as a static message.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -43,4 +43,39 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
```
Comment (parvathy-sudhir-pillai) [inline test/java/seedu/addressbook/common/UtilsTest.java:48]: This commit shouldn't be part of this PR. In general, a PR should not contain unmerged code from other PRs. Try to limit each PR to one enhancement, which means the PR should be based on a branch that starts off from the master branch that is in sync with the upstream master branch

**Review by parvathy-sudhir-pillai — APPROVED** [review]
Good effort on this PR @lywjoel. In particular, good job on,
Suggestions to improve further:
* update user and developer docs
* update I/O and jUnit tests to test the new functionality
* limit the PR to one enhancement and avoid including changes not related to that enhancement or unmerged code from other PRs
* Pay closer attention to the coding standard

Please close the PR after viewing comments.
